publish/prepare-artifacts.proj

```diff
@@ -137,6 +137,57 @@
       Importance="High" />
   </Target>
 
+    <Target Name="PreparePublishFilesToAzureBlobFeed"
+          DependsOnTargets="GetProductVersions">
+    <PropertyGroup>
+
+      <AssetManifestFilename>Manifest.xml</AssetManifestFilename>
+      <AssetManifestFile>$(ArtifactsLogDir)AssetManifest/$(AssetManifestFilename)</AssetManifestFile>
+
+      <!-- Create temp dir to store generated asset manifest, per Arcade guidance. -->
+      <TempWorkingDir>$(ArtifactsObjDir)TempWorkingDir\$([System.Guid]::NewGuid())\</TempWorkingDir>
+
+      <ProductVersionTxtContents Condition="'$(StabilizePackageVersion)'=='true'">$(ProductionVersion)</ProductVersionTxtContents>
+      <ProductVersionTxtContents Condition="'$(StabilizePackageVersion)'!='true'">$(ProductVersion)</ProductVersionTxtContents>
+
+    </PropertyGroup>
+
+    <!-- Generate windowsdesktop-productVersion.txt containing the value of $(PackageVersion) -->
```

Comment:
Let’s have it generate both windowsdesktop-productVersion.txt and productVersion.txt. So duplicate lines 156-160, changing the file name, and 165-167.